style.css

```diff
@@ -36,6 +36,38 @@ section {
   flex-grow: 1;
 }
 
+.navigator ul {
+  list-style: none;
+  display: flex;
+  text-decoration: none;
+}
```

Comment:
- [ ] As requested by the previous reviewer, It's a nice practice to add colour(eg: `ul a { color: #344563 }`) to your links to stick the links from the Figma template.
- [ ] OPTIONAL: For the logo, it's a good idea if you increase its `font-weight` to match the logo design.
1. ![image](https://user-images.githubusercontent.com/80612925/172893990-9747ab89-34b1-4356-8f23-ea6deda0826e.png)
2. Figma design
![image](https://user-images.githubusercontent.com/80612925/172894451-fbd8c76a-8c31-484e-9507-47246ae78b15.png)